learn/sherif/main_test/src/main/java/frc/robot/io/Slider.java

```diff
@@ -7,36 +7,45 @@
 import java.util.Map;
 
 public class Slider {
-    private final SimpleWidget m_value;
+    private final SimpleWidget value;
+    private double lastDouble;
+
     public Slider(String name, double defaultValue) {
         this(name, defaultValue, 0, 1.0);
     }
 
     public Slider(String name, double defaultValue, double min, double max) {
-        m_value = BoardManager.getManager().getTab()
+        value = BoardManager.getManager().getTab()
                 .add(name, defaultValue)
                 .withWidget(BuiltInWidgets.kNumberSlider)
                 .withProperties(Map.of("min", min, "max", max));
+        this.lastDouble = defaultValue;
```

Comment:
wouldn't it make sense to set the value of the network entry to defaultValue as well?